packages/react-components/react-nav-preview/stories/NavDrawer/NavDrawer.stories.tsx

```diff
@@ -8,32 +8,63 @@ import {
   NavSubItem,
   NavSubItemGroup,
 } from '@fluentui/react-nav-preview';
-import { DrawerBody } from '@fluentui/react-drawer';
-import { makeStyles, shorthands, tokens } from '@fluentui/react-components';
+import { DrawerBody, DrawerProps } from '@fluentui/react-drawer';
+import { Button, Label, Radio, RadioGroup, makeStyles, shorthands, tokens, useId } from '@fluentui/react-components';
 import { Folder20Filled, Folder20Regular, bundleIcon } from '@fluentui/react-icons';
+
 const useStyles = makeStyles({
   root: {
+    ...shorthands.border('2px', 'solid', '#ccc'),
     ...shorthands.overflow('hidden'),
+
     display: 'flex',
-    height: '480px', // arbitrary value, for demo purposes only
-    // arbitrary value, for dramatic effect only
-    // I know this is ugly and wrong, but styling is coming, I promise.
-    backgroundColor: tokens.colorBackgroundOverlay,
+    height: '480px',
+    backgroundColor: '#fff',
+  },
+
+  content: {
+    ...shorthands.flex(1),
+    ...shorthands.padding('16px'),
+
+    display: 'grid',
+    justifyContent: 'flex-start',
+    alignItems: 'flex-start',
+    gridRowGap: tokens.spacingVerticalXXL,
+    gridAutoRows: 'max-content',
+  },
+
+  field: {
+    display: 'grid',
+    gridRowGap: tokens.spacingVerticalS,
   },
 });
 
 const Folder = bundleIcon(Folder20Filled, Folder20Regular);
 
-export const Default = (props: Partial<NavDrawerProps>) => {
+type DrawerType = Required<DrawerProps>['type'];
+
+export const NavDrawerDefault = (props: Partial<NavDrawerProps>) => {
```

Comment:
Any reason to rename the file? We usually try to keep the name as `<component><story goal>` in the case for default `<Component>Default`

Reply:
I think I was just tidying some file conflicts. Will rename back.